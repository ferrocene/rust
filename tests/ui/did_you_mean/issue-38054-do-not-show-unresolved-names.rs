//@ revisions: edition2015 edition2018
//@ [edition2015] edition: 2015
//@ [edition2018] edition: 2018..
use Foo; //~ ERROR unresolved

use Foo1; //~ ERROR unresolved

fn main() {}
