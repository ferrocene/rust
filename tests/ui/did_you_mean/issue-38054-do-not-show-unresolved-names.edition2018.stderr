error[E0432]: unresolved import `Foo`
  --> $DIR/issue-38054-do-not-show-unresolved-names.rs:4:5
   |
LL | use Foo;
   |     ^^^ no external crate `Foo`

error[E0432]: unresolved import `Foo1`
  --> $DIR/issue-38054-do-not-show-unresolved-names.rs:6:5
   |
LL | use Foo1;
   |     ^^^^ no external crate `Foo1`

error: aborting due to 2 previous errors

For more information about this error, try `rustc --explain E0432`.
